testlib/src/main/java/com/diffplug/spotless/ResourceHarness.java

```diff
@@ -42,6 +42,12 @@
 import com.diffplug.common.io.Resources;
 
 public class ResourceHarness {
+
+	/**
+	 * default test testPath.
+	 */
+	protected String testPath = "src/main/java/test.java";
```

Comment:
Mutable state is bad. Before this PR, it might be possible to enable method-level parallelism, after this PR it is definitely impossible. For Spotless we prefer functional style whenever possible.

Reply:
agree, lets make the field final.